template.yaml

```diff
@@ -642,6 +642,39 @@ Resources:
             Type: SQS
             Destination: !GetAtt PublicationFanoutHandlerDLQ.Arn
 
+    DeleteDraftPublicationHandlerDLQ:
+      Type: "AWS::SQS::Queue"
+
+    DeleteDraftPublicationHandler:
+      Type: AWS::Serverless::Function
+      Properties:
+        Layers:
+          - !Ref NvaPublicationCommonsLayer
+        CodeUri: delete-draft-publication-handler
+        Handler: no.unit.nva.publication.delete.DeleteDraftPublicationHandler::handleRequest
+        Runtime: java11
+        MemorySize: 512
+        Role: !GetAtt LambdaRole.Arn
```

Comment:
You have to create a new role which allows the publication to delete items from the dynamo table.